ts/screens/profile/useContentWithFF.tsx

```diff
@@ -0,0 +1,27 @@
+import { useCallback } from "react";
+import _ from "lodash";
+import { TranslationKeys } from "../../../locales/locales";
+import I18n from "../../i18n";
+import { useIOSelector } from "../../store/hooks";
+import { isSettingsVisibleAndHideProfileSelector } from "../../store/reducers/backendStatus";
+
+const useContentWithFF = (i18nString: TranslationKeys) => {
+  const isSettingsVisibleAndHideProfile = useIOSelector(
+    isSettingsVisibleAndHideProfileSelector,
+    _.isEqual
+  );
+
+  const getContentWithFF = useCallback(
+    (i18nString: TranslationKeys, selector: boolean) =>
```

Comment:
The second parameter might be more explicit, I think the name `selector` is too generic. What do you think about changing its name in `isSettingsVisible` or something like that?

Reply:
I didn't give specific names because I would like to keep this custom hook generic, so that everyone can use it. An alternative generic name could be 'discriminationValue'. What do you think?

Reply:
yes, thanks for this suggestion!